src/main/java/com/google/codeu/data/User.java

```diff
@@ -5,11 +5,13 @@
   private String email;
   private String aboutMe;
   private String profilePicUrl;
+  private String nickname;
 
-  public User(String email, String aboutMe, String profilePicUrl) {
+  public User(String email, String aboutMe, String profilePicUrl, String nickname) {
```

Comment:
Wondering if there is a better way to do this constructor for `profilePicUrl` and `nickname`. Seems like the logic for checking these two parameters are done before calling this constructor in the `AboutMeServlet` and `ImageHandlerServlet`. I think either making these two parameters optional in this constructor (and then defining the default values in here) or overloading this constructor might be neater. Just a suggestion to be explored, doesn't have to be done that way.

Reply:
Hmm I see, I'll explore more about this but I think I'll merge the PR first and make another PR with the changes, thanks!